apps/omg_child_chain/lib/omg_child_chain/block_queue/core.ex

```diff
@@ -340,6 +351,15 @@ defmodule OMG.ChildChain.BlockQueue.Core do
     :ok
   end
 
+  # Fallback for unknown server errors: https://eth.wiki/json-rpc/json-rpc-error-codes-improvement-proposal
+  # Only server errors are handled as they are the only set of errors that we have no control of.
+  # Returns `:ok` so that BlockQueue can continue and do a retry, similar to a low replacement price error.
+  def process_submit_result(submission, {:error, %{"code" => code} = error}, _newest_mined_blknum)
+      when code >= -32_099 and code <= -32_000 do
+    _ = Logger.error("Submission #{inspect(submission)} resulted in an unknown server error: #{inspect(error)}.")
```

Comment:
Do we alarm on all error log. If not let's make sure we add alarm on this, either via Datadog parsing the error message or do the direct alarm (?) from code?

Reply:
Good idea! I think we can do with an app alarm when a block submission is pending for more than 2 blocks (since we're aiming for next-block inclusion, but I'll make number configurable in case of too many false panics). Then Datadog can consume this app alarm similar to `:ethereum_stalled_sync` and friends.

Adding now to this PR.

Reply:
> Since we're losing this visibility because this PR is suppressing the exception, I'll add :block_submission_stalled alarm to this PR.


Queue observability size. Hell yeah!